joss/paper.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'MParT: Monotone Parameterization Toolkit'
+tags:
+  - Python
+  - Julia
+  - Matlab
+  - c++
+  - measure transport
+  - transport map
+  - density estimation
+  - Bayesian inference
+  - normalizing flows
+authors:
+  - name: Matthew Parno
+    orcid: 0000-0002-9419-2693
+    corresponding: true
+    affiliation: 1
+  - name: Paul-Baptiste Rubio
+    orcid: 0000-0002-9765-1162
+    affiliation: 2
+  - name: Daniel Sharp
+    orcid: 0000-0002-0439-5084
+    affiliation: 2
+  - name: Michael Brennan
+    orcid: 0000-0001-7812-9347
+    affiliation: 2 
+  - name: Ricardo Baptista 
+    orcid: 0000-0002-0421-890X
+    affiliation: 2
+  - name: Henning Bonart
+    orcid: 0000-0002-5026-4499
+    affiliation: 2
+  - name: Youssef Marzouk 
+    orcid: 0000-0001-8242-3290
+    affiliation: 2
+affiliations:
+ - name: Dartmouth College, USA
+   index: 1
+ - name: Massachusetts Institute of Technology, USA
+   index: 2
+date: 22 July 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Measure transport is a rich area in applied mathematics that involves the construction of deterministic transformations---known as transport maps---between probability distributions [@santambrogio2015optimal]. These maps characterize a complex target distribution as a (deterministic) transformation of a simple reference distribution (e.g., a standard Gaussian). In the context of probabilistic modeling, transport maps enable easy generation of samples from the target distribution and direct evaluation of the target probability density function. Monotone triangular maps [@baptista2022adaptive] are specific class of transport maps endowed with several computational advantages over non-triangular maps, and yet sufficiently general to represent any absolutely continuous distribution; they are also the building block of many normalizing flow architectures commonly used in the machine learning community [@papamakarios2021normalizing].
+
+Triangular maps are also well suited to many tasks in Bayesian inference, including the modeling of conditional distributions [@Marzouk2016,@spantini2018inference] and the acceleration of posterior sampling [@el2012bayesian; @bigoni2016adaptive; @parno2018transport; @cotter2019ensemble].  The fundamental idea is to convert the problem of characterizing a probability distribution through Monte Carlo sampling, variational inference, or density estimation into an optimization problem over multivariate monotone functions. The efficient solution of this optimization problem is important when using maps as part of online algorithms, as commonly encountered in sequential inference [@spantini2019coupling].
```

Comment:
deterministic optimization problem? Or even add more specific attributes like convex (is it a convex problem?)?

Reply:
The optimization can be convex (in the case of samples) or not (in the case of densities).  It can also be deterministic (e.g., using all the samples in a sample average approximation formulation like we're doing now) or stochastic (e.g., if we used mini batches and stochastic approximation techniques).   I'm going to leave this one as is for now.